### PR TITLE
make wave component optional in s2s 35 day test #142

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
I've tested running 1 hour for the 35d tests. Should I submit the whole 35d test for the weekend?
#142 
new file:   tests/fv3_conf/cpld_fv3_mom6_cice_bmark_35d_run.IN
modified:   tests/rt.sh
new file:   tests/rt_35d_noww3_cmeps.conf
new file:   tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d
new file:   tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_cold

### CODE REVIEW

File: tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+export TPN=40
+export INPES=6
+export JNPES=12
+export THRD=1
+export WRTTASK_PER_GROUP=24
+
+export med_petlist_bounds="0 431"
+export atm_petlist_bounds="0 455"
+export ocn_petlist_bounds="456 695"
+export ice_petlist_bounds="696 743"
```
Comment (DeniseWorthen) [inline tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: You'll want this to be the same as in your cold test, where you specify the cpl_bmark settings.

File: tests/rt.sh [hunk 2]
```diff
@@ -425,7 +425,7 @@ fi
 if [[ $MACHINE_ID = cheyenne.* ]]; then
   RTPWD=${RTPWD:-$DISKNM/develop-20200210/${COMPILER^^}}
 else
-  RTPWD=${RTPWD:-$DISKNM/FV3-MOM6-CICE5/develop-20200718}
+  RTPWD=${RTPWD:-$DISKNM/FV3-MOM6-CICE5/develop-20200530}
```
Comment (DeniseWorthen) [inline tests/rt.sh]: why isn't this the same date as the develop branch?

Comment (BrianCurtis-NOAA) [reply tests/rt.sh]: I think @MinsukJi-NOAA said the 35d files were being held there, but i don't think thats being used. So we can keep the current date/time.

File: tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d (hunk 1 above)
Comment (BrianCurtis-NOAA) [reply tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d]: in the 35d/cold with ww3, this is how it was as they are just copies of the two files edited to remove the WW3/wav variables/exports.
So the cold and 35d both need the dynamic variables and not hard coded values?
Should this also be true in the files with WW3?

File: tests/rt.sh (hunk 2 above)
Comment (MinsukJi-NOAA) [reply tests/rt.sh]: Yeah, to run 35d 'warm' start, we need to point to 20200530 for mediator data. But it should be changed back to the latest when committing the codes.

Comment (BrianCurtis-NOAA) [reply tests/rt.sh]: OK i'll get that back to the old one!

**Review by DeniseWorthen — APPROVED** [review]
Looks good to me.

**Review by binli2337 — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]
The code changes look good.  I assume the code changes are tested with 35 days run with and without ww3.

**Comment by BrianCurtis-NOAA** [comment]
> 
> 
> The code changes look good. I assume the code changes are tested with 35 days run with and without ww3.

The 35d with WW3 didn't change. You just choose the 35d_noww3 or just 35d. I ran the 35d_noww3 test.

**Review by MinsukJi-NOAA — APPROVED** [review]
Looks good to me

**Pull request merged**